.github/workflows/docker-image.yml

```diff
@@ -60,12 +60,11 @@ jobs:
       - name: Build Base Docker Image
         id: docker_build
         uses: docker/build-push-action@v4
-        if: github.event_name == 'push' && github.ref != 'refs/heads/master'
         with:
           context: ./
           file: ./Dockerfile
           builder: ${{ steps.buildx.output.name }}
-          push: true
+          push: ${{ github.event_name == 'push' && github.ref != 'refs/heads/master' }}
```

Comment:
What's the motivation for this change? Does it change anything functionally?

Reply:
Looks like it only pushes on tags rather than every commit?

Reply:
It got changed from being in the 'if' block to down here. Should have put my comment on the range so that the diff showed up better. Just curious the motivation.
![image](https://user-images.githubusercontent.com/187133/231050376-96db5d58-0815-4f8e-ac2a-ffa0764ee867.png)

Reply:
it doesn't build docker image with previous `if` field， dockerfile is not tested in the pr

Reply:
it doesn't actually change anything， previous ci only push and build on the condition， now it build in any condition， and push in the condition

Reply:
https://github.com/Flexget/Flexget/actions/runs/4659863819/jobs/8247244212

Reply:
Ahh, I get it now. Sounds good!